### PR TITLE
Add FreeBSD "instructions" for making a release

### PR DESCRIPTION
**Pull request by smortex**
### Description

There is not much to do on the OpenSearch side, but document it so that
it can fit in some workflow.

### Issues Resolved
https://github.com/opensearch-project/opensearch-build/issues/101#issuecomment-922844301
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by smortex** [comment]
Cc @dblock who asked for this :grin:

File: README.md [hunk 1]
```diff
@@ -30,6 +30,12 @@ OpenSearch and its components are built from source, assembled, signed and teste
 
 TODO
 
+#### Releasing for FreeBSD
```
Comment (dblock) [inline README.md:33]: Add to TOC, please.

File: README.md [hunk 2]
```diff
@@ -30,6 +30,12 @@ OpenSearch and its components are built from source, assembled, signed and teste
...
+
+The FreeBSD ports and packages for OpenSearch are managed by an *[OpenSearch Team](https://wiki.freebsd.org/OpenSearch)* at FreeBSD.  When a new release is rolled out, this team is in charge for updating the port and committing it to the FreeBSD ports tree. Anybody is welcome to help the team by providing patches for [upgrading the ports](https://docs.freebsd.org/en/books/porters-handbook/book/#port-upgrading) following the [FreeBSD Porter's Handbook](https://docs.freebsd.org/en/books/porters-handbook/book/) instructions.
```
Comment (dblock) [inline README.md:35]: Let's say "by a community OpenSearch team", we want to give credit to the fact that it's not maintained by an amazon team

Comment (dblock) [inline README.md:35]: "is in charge" -> "will"? Otherwise it reads like a paid job

File: README.md [hunk 3]
```diff
@@ -30,6 +30,12 @@ OpenSearch and its components are built from source, assembled, signed and teste
...
+
+Make sure the OpenSearch Team is aware of the release by droping a mail to opensearch@FreeBSD.org.
```
Comment (dblock) [inline README.md:37]: I would just remove this or replace by saying "open an issue in xyz repo". Email addresses get notoriously spammy and you already mention it in the page you link.

**Review by dblock — CHANGES_REQUESTED** [review]
Thanks. Fix TOC is the only must, but please consider the rest of my suggestions.

File: README.md (hunk 1 above)
Comment (smortex) [reply README.md:34]: Done!

File: README.md (hunk 2 above)
Comment (smortex) [reply README.md]: Done!

Comment (smortex) [reply README.md]: Done!  As a non-native English speaker both where equivalent :upside_down_face:

File: README.md (hunk 3 above)
Comment (smortex) [reply README.md]: Removed!

**Comment by smortex** [comment]
Commit updated as requested.

Thanks!

**Review by dblock — APPROVED** [review]

**Review by saratvemulapalli — APPROVED** [review]

**Pull request merged**